src/main/java/net/dv8tion/jda/api/events/guild/member/GuildMemberUpdateNicknameEvent.java

```diff
@@ -40,18 +41,48 @@ public GuildMemberNickChangeEvent(JDA api, long responseNumber, Member member, S
      *
      * @return The old nickname
      */
-    public String getPrevNick()
+    public String getOldNickname()
     {
-        return prevNick;
+        return getOldValue();
     }
 
     /**
      * The new nickname
      *
      * @return The new nickname
      */
-    public String getNewNick()
+    public String getNewNickname()
+    {
+        return getNewValue();
+    }
+
+    @Override
+    public String getPropertyIdentifier()
+    {
+        return "nick";
```

Comment:
This should be a constant in this class, compare to other classes that implement the `UpdateEvent` interface. Additionally it should be in the class-level docs.